### PR TITLE
Update delete db

### PR DESCRIPTION
**Pull request by getssh**
Hey, I have worked on the query and update animals table
 - Added more data to the animals table
 - Deleted some row from the animals table following the instruction
 - Updated animals table
 - Write queries to answer questions

**Note**
I have included the screenshots that show the written queries and their output.
I used one screenshot to show multiple outputs, for example, for the following transaction:-

> Inside a transaction:
>   - Delete all animals born after Jan 1st, 2022.
>   - Create a savepoint for the transaction.
>   - Update all animals' weight to be their weight multiplied by -1.
>   - Rollback to the savepoint
>   - Update all animals' weights that are negative to be their weight multiplied by -1.
>   - Commit transaction

I used one screenshot.
 
- update the data.sql with the newly added data query

output1

output2

output3

output4

output5

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -5,3 +5,14 @@ VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23),
        ('Gabumon', '2018-11-15', 2, TRUE, 8),
        ('Pikachu', '2021-01-07', 1, FALSE, 15.04),
        ('Devimon', '2017-05-12', 5, TRUE, 11);
+
+/* Add more data to animals table */
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, FALSE, -11),
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'-11.0'`

File: data.sql [hunk 2]
```diff
@@ -5,3 +5,14 @@ VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23),
...
+       ('Plantmon', '2021-11-15', 2, TRUE, -5.7),
+       ('Squirtle', '1993-04-02', 3, FALSE, -12.13),
+       ('Angemon', '2005-06-12', 1, TRUE, -45),
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'-45.0'`

File: data.sql [hunk 3]
```diff
@@ -5,3 +5,14 @@ VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23),
...
+       ('Boarmon', '2005-06-07', 7, TRUE, 20.4),
+       ('Blossom', '1998-10-13', 3, TRUE, 17),
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'17.0'`

File: data.sql [hunk 4]
```diff
@@ -5,3 +5,14 @@ VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23),
...
+       ('Ditto', '2022-05-14', 4, TRUE, 22);
```
Comment (zieeco) [inline data.sql]: - Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'22.0'`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @getssh,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice schema file. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @getssh**
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Clean Codebase
✔️ Great job implementing the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**